apps/docs/content/components/forms/PasswordField.mdx

```diff
@@ -1,48 +1,89 @@
 ---
-title: PasswordField (WIP)
-description: TBD
+title: PasswordField
+description: A specialized text field that shows/hides a password.
 category: "forms"
 links:
     source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/inputs/src/PasswordField.tsx
 ---
 
+<Example src="inputs/docs/passwordField/preview" isOpen />
+
 ## Props
 
-TODO: Add the props table, if we have multiple props table (ex: TagList and Tag) in the same file,  use the name of the component in ### before the each props table
+<PropTable component="PasswordField" />
+
+## Anatomy
 
-## Guidelines
+<FeatureFlag flag="next">
+    ## Guidelines
 
-TODO: If we have some guidelines about this component's usage
+    TODO: If we have some guidelines about this component&apos;s usage
 
-### Accessibility ?
+    ### Accessibility?
 
-TODO: If we have some guidelines about this component and accessibility
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
 
-## Anatomy
+<FeatureFlag flag="rc">
 
-TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
 
-### Concepts
+    ### Concepts
 
-TODO: links to related concepts
+    TODO: links to related concepts
+</FeatureFlag>
 
 ### Composed Components
 
-TODO: links to related components (most likely all component which are used by the slots)
+A `PassworField` uses the following components.
+
+- [Label](./Label)
+- [Helper Message](./HelperMessage)
+- [Error Message](./ErrorMessage)
 
 ## Examples
 
-## Advanced customization
+### Disabled
+A password field in a disabled state shows that an input field exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that a field may become available later.
+
+<Example src="inputs/docs/passwordField/disabled" />
+
+### ReadOnly
+The `isReadOnly` prop makes the PasswordField's text content immutable. Unlike `isDisabled`, the PasswordField remains focusable and the contents can still be copied. See [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) for more information.
+
+<Example src="inputs/docs/passwordField/readOnly" />
+
+### Error
+A password field with an error message.
+
+<Example src="inputs/docs/passwordField/error" />
+
+### Sizes
+Password fields come in two sizes.
```

Comment:
I would be more generic, like in other size component docs, this way if we add a third size, nothing to change here :D